airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/parquet/JsonSchemaConverterTest.java

```diff
@@ -0,0 +1,84 @@
+package io.airbyte.integrations.destination.s3.parquet;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.common.collect.Lists;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.commons.resources.MoreResources;
+import io.airbyte.commons.util.MoreIterators;
+import java.util.Collections;
+import java.util.stream.Stream;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtensionContext;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.ArgumentsProvider;
+import org.junit.jupiter.params.provider.ArgumentsSource;
+
+class JsonSchemaConverterTest {
+
+  @Test
+  public void testGetTypes() {
+    // single type
```

Comment:
can we make these separate cases and test every type? might need to be parametrized

Reply:
I have split them. They are just two assertions. Let's not overcomplicate this by parameterizing it.